### PR TITLE
sd-nelink prettification

### PR DESCRIPTION
**Pull request by keszybz**
After the next realase.

### CODE REVIEW

**Review by bluca — APPROVED** [review]
lgtm

**Comment by yuwata** [comment]
LGTM too. 

> After the next realase.

I cannot find no reason to postpone this...

**Comment by keszybz** [comment]
OK, let's merge then.

File: src/libsystemd/sd-netlink/netlink-socket.c [hunk 1]
```diff
@@ -64,23 +63,15 @@ static int broadcast_groups_get(sd_netlink *nl) {
...
+                                unsigned group = i * sizeof(uint32_t) * 8 + j + 1;
 
-                        r = hashmap_put(nl->broadcast_group_refs, UINT_TO_PTR(group), UINT_TO_PTR(1));
-                        if (r < 0)
-                                return r;
-                }
-        }
+                                r = hashmap_put(nl->broadcast_group_refs, UINT_TO_PTR(group), UINT_TO_PTR(1));
+                                if (r < 0)
+                                        return r;
```
Comment (poettering) [inline src/libsystemd/sd-netlink/netlink-socket.c:73]: hmm, i'd really invert the if check and continue. deep indentation sucks

**Pull request merged**